### DISCUSSION TITLE
How to Relink my downloaded project file from my github AC represatory to same represatory

### DISCUSSION THREAD

**Discussion by ShuvaMallickPro** [community/community#47145] [Repositories]
### Select Topic Area

Product Feedback

### Body

All my project files have been removed due to my hard disk corruption. I am downloading my project from my GitHub account again.. I want my downloaded project to commit/re-establish again like my previous project link into my GitHub repository, Stay previous all commits and others record history. How can I do this? Could you plz help me?

**Comment by P123838** [comment]
好
